velox/dwio/dwrf/reader/SelectiveStructColumnReader.cpp

```diff
@@ -61,6 +61,8 @@ SelectiveStructColumnReader::SelectiveStructColumnReader(
         FlatMapContext{encodingKey.sequence, nullptr}));
     childSpec->setSubscript(children_.size() - 1);
   }
+  thread_local std::string readerDebugString;
+  debugString_ = readerDebugString;
```

Comment:
Why do we need debugString_ instead of just using readerDebugString? Also, it looks like we don't initialize readerDebugString? (Maybe I didn't understand thread_local. Could you explain a bit about this?)